examples/README.md

```diff
@@ -0,0 +1,251 @@
+# Examples
+
+Here is a collection of examples of `Synapse` resources that illustrate the
+main features of the Synapse operator.
+
+## Pre-requisites
+
+The Synapse operator should be running in your cluster. See the
+[deployment intructions](https://github.com/opdev/synapse-operator#deploying-the-synapse-operator).
+
+## Your first Synapse deployment
+
+A minimal manifest example of a `Synapse` resource can be found under the
+`01-my-first-synapse-deployment/` folder. To deploy Synapse, simply run:
+
+```shell
+$ kubectl apply -f 01-my-first-synapse-deployment/synapse.yaml
+synapse.synapse.opdev.io/my-first-synapse-deployment created
+```
```

Comment:
Maybe examples/01-my-first-synapse-deployment/synapse.yaml
It seems obvious but that people will bump into that for sure. Having the examples/ for the path on those files helps. Or explicitly tell them to cd examples because after cloning the project they will be at the root folder.

Reply:
Ok, I wasn't sure about that one. Thanks for the feedback, I'll change it.